dts/x86/intel_curie.dtsi

```diff
@@ -183,5 +183,14 @@
 			label = "WATCHDOG_0";
 
 		};
+
+		usbd: usbd@000a0000 {
```

Comment:
is this correct for this platform? QMSI has a different value.

```
ext/hal/qmsi/soc/quark_se/include/qm_soc_regs.h:#define QM_USB_0_BASE (0xB0500000)
```

Reply:
done